### ISSUE TITLE
Starring repo

### ISSUE THREAD

**Issue by pcr910303** [GitHawkApp/GitHawk#2656]
Firstly, I started to use this Github client a month ago, and it’s fantastic! Thanks for this great app.
I usually keep track of interesting repos by starring them. It looks like that showing all starred repos will go to the WIP profile page.
However, I’m not sure how to star a repo I visited in GitHawk. I find myself copying the URL of the repo and visiting in Safari...
A star button on the top will be sufficient IMHO...(Or a better UX will be great!)

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by pcr910303** [comment]
Oh and watching too!

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by BasThomas** [comment]
Hey @pcr910303, thanks for bringing this up! Starring a repository is currently not supported, but tracked by #397. Feel free to chime in there.

> I find myself copying the URL of the repo and visiting in Safari...

In the meantime, you can always open Safari from the "..." menu, which might be easier than copying the URL.

> Oh and watching too!

Sounds good!

**Comment by BasThomas** [comment]
> I started to use this Github client a month ago, and it’s fantastic! Thanks for this great app.

Thanks for the kind words! 😍

**Comment by Huddie** [comment]
@BasThomas is this solved in PR #2482 ?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by BasThomas** [comment]
I don’t think so. This is about watching a _repository_ rather than managing subscriptions to _issues and pull requests_

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by Huddie** [comment]
Ah okay got it!

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>
